config/pKeyAssociations.json

```diff
@@ -0,0 +1,6 @@
+{
+  "User": "username",
+  "Application": "name",
+  "UserGroup": "name",
+  "Layer": "name"
```

Comment:
Hmm. Is there a reason not to use a numeric key? I can imagin people having multiple layers with the same name at least...

Reply:
This association table specifies which property will be displayed in `Select` etc.

Reply:
ok, stupid me